server/helpers/paginationHelper.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * @name paginationValues
+ * @description function that returns offset and limit for pagination
+ * @param {Object} query object with page and pageItems as integers
+ * @returns {(offset|limit)} offset and limit to pagenate request
+ */
+const paginationValues = ({ page = 1, pageItems = 10 }) => {
+  const offset = (page - 1) * pageItems;
+  const limit = offset + +pageItems;
```

Comment:
what's happening here?

Reply:
`pageItems` is coerced to a number by the unary operator because it comes as a string from `req.query`. Then it is added to offset to get the number of items to return. The other option for converting it to a number is using this: `Number(pageItems)`. I chose the unary operator because it is more concise.

Reply:
I didn't add the unary operator to the calculation for offset because JS will actually treat it as a number there. However, when you try to add, it does string concatenation rather than an addition. That is why it is necessary for the addition calculation

Reply:
please make it an integer before proceeding with any other statement